packages/accounts-passwordless/passwordless_server.js

```diff
@@ -114,7 +114,7 @@ function generateSequence() {
 }
 
 Meteor.methods({
-  requestLoginTokenForUser: ({ selector, userData, options = {} }) => {
+  async requestLoginTokenForUser({ selector, userData, options = {} }) {
```

Comment:
Changing this method to async may need to also use Meteor.callAsync where is used, in the client part. So here may need to also be adapted, https://github.com/harryadel/meteor/blob/fix/_checkForCaseInsensitiveDuplicates/packages/accounts-passwordless/passwordless_client.js#L106-L110

cc @Grubba27

Reply:
To avoid breaking the API now, we could use the callAsync version and the callback to resolve the issue when the promise is resolved.

And may add the possibility of returning a promise for Accounts.requestLoginTokenForUser.

People can still use the callback-based API or resolve the API using promises.

```js

Accounts.requestLoginTokenForUser(options, (err) => err ? "ok" : "not ok"
)

/// could be used as well like this:

try {
  await Accounts.requestLoginTokenForUser(options);
  "ok"
} catch (err) {
  "not ok"
}


```

It would not be a breaking change, but rather a new feature in this PR

Reply:
`Meteor.call`/`Meteor.apply` work fine when the method is async on the server. With Meteor 2, it's only a problem if the stub is async.

Reply:
Ok. Thank you @zodern.

So, [in this example](https://github.com/harryadel/meteor/blob/fix/_checkForCaseInsensitiveDuplicates/packages/accounts-passwordless/passwordless_client.js#L106-L110) if I am not mistaken, the method is [server-only as described on server file only](https://github.com/harryadel/meteor/blob/fix/_checkForCaseInsensitiveDuplicates/packages/accounts-passwordless/passwordless_server.js#L117), and Meteor.call/apply would still work fine when called from the client.

I think the same happens for this exact code on Meteor 3.x, where is async actually and server-only.

Reply:
What's the best course of action here? Keep it as is? Or change it use to `.callAsync`?

Reply:
I think we can leave it as it is, since is server-only. So I will skip any adoption here according of what mentioned above. 

Even in Meteor 3.x [is async on server](https://github.com/meteor/meteor/blob/release-3.0/packages/accounts-passwordless/passwordless_server.js#L117) and called with [call on the client](https://github.com/meteor/meteor/blob/release-3.0/packages/accounts-passwordless/passwordless_client.js#L106-L110). So lets skip this comment here.